test/Header/AddressListHeaderTest.php

```diff
@@ -210,4 +210,27 @@ public function getAddressListsWithGroup()
             ['To: friends: john@example.com; enemies: john@example.net, bart@example.net;', 3, 'john@example.net'],
         ];
     }
+
+    public function specialCharHeaderProvider()
+    {
+        return [
+            ["To: =?UTF-8?B?dGVzdCxsYWJlbA==?= <john@example.com>, john2@example.com", ['john@example.com' => 'test,label', 'john2@example.com'], 'UTF-8'],
```

Comment:
This is not legible. Split in different lines. Also I suggest assert exactly the same unencoded value.